cypress/integration/calculator.spec.js

```diff
@@ -0,0 +1,247 @@
+/// <reference types="cypress" />
+import CalculatorPage, { Operations } from "../support/page_objects/calculator";
+
+const calculator = new CalculatorPage();
+
+const builds = Array.from({ length: 10 }, (x, i) => i.toString()); // Make array of strings from "0" to "9"
+
+const numberPairs = [
+  {
+    num1: 5.15,
+    num2: 5.599,
+  },
+  {
+    num1: 0,
+    num2: 53,
+  },
+  {
+    num1: -128,
+    num2: -543,
+  },
+  {
+    num1: 1.5,
+    num2: -12.8,
+  },
+  {
+    num1: -14,
+    num2: 58.624,
+  },
+];
+
+const invalidNumberPairs = [
+  {
+    num1: "a",
+    num2: 1.2,
+  },
+  {
+    num1: 2.1,
+    num2: "b",
+  },
+  {
+    num1: "a",
+    num2: "b",
+  },
+];
+
+// TODO: Move repetitive test code to reusable functions
+describe("Basic calculator", () => {
+  beforeEach(() => {
+    calculator.visit();
+  });
+
+  builds.forEach((build) => {
+    /**
+     * Happy path scenario
+     */
+    describe(`Build ${build} - HP`, () => {
```

Comment:
I would suggest not to use _describe_ in _describe_. Maybe change one of them to _context_? Or even rethink different structure? 
A little bit about test organizing: https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Test-Structure
Context: https://www.codota.com/code/javascript/functions/mocha/context

Reply:
Thanks for the suggestion!